apps/balancer-tools/src/app/poolsimulator/(components)/PoolParamsForm.tsx

```diff
@@ -38,19 +38,29 @@ const docsMapper = {
   [PoolTypeEnum.Fx]: "https://docs.xave.co/product-overview-1/fxpools",
 };
 
+const dynamicGetMinMaxAndStep = {
+  getMin: (n?: number) => (n ? n / 100 : 0.01),
+  getMax: (n?: number) => (n ? n * 10 : 0.01),
+  getStep: (n?: number) => (n ? n / 100 : 0.01),
+} as const;
+
+const SwapFeeInput = {
+  name: "swapFee",
+  label: "Swap Fee",
+  unit: "%",
+  transformFromDataToForm: (n?: number) =>
+    typeof n === "number" ? n * 100 : undefined,
+  transformFromFormToData: (n?: number) =>
+    typeof n === "number" ? n / 100 : undefined,
+  tooltip: "Percentage fee charged on swaps",
+  getMin: (_n?: number) => 0,
+  getMax: (_n?: number) => 10,
+  getStep: (_n?: number) => 0.001,
+} as const;
+
 const inputMapper = {
   [PoolTypeEnum.MetaStable]: [
-    {
-      name: "swapFee",
-      label: "Swap Fee",
-
-      unit: "%",
-      transformFromDataToForm: (n?: number) =>
-        typeof n === "number" ? n * 100 : undefined,
-      transformFromFormToData: (n?: number) =>
-        typeof n === "number" ? n / 100 : undefined,
-      tooltip: "Percentage fee charged on swaps",
-    },
+    SwapFeeInput,
```

Comment:
what did we gain by moving this to a variable? also assigning PascalCase consts it's not the pattern we're using

Reply:
I think that is easier to update (if necessary). This field is exactly the same for all pools.